trieste/acquisition/multi_objective/pareto.py

```diff
@@ -73,6 +81,135 @@ def hypervolume_indicator(self, reference: TensorType) -> TensorType:
         )
         return hypervolume_indicator
 
+    def sample(self, sample_size: int) -> Tuple[TensorType, TensorType]:
```

Comment:
Since we have `from __future__ import annotations`, you can (and should) do this instead.

```suggestion
    def sample(self, sample_size: int) -> tuple[TensorType, TensorType]:
```